pennylane/transforms/decompositions/clifford_t_transform.py

```diff
@@ -371,7 +371,7 @@ def circuit(x, y):
 
     with QueuingManager.stop_recording():
         # Build the basis set and the pipeline for initial compilation pass
-        basis_set = [op.__name__ for op in _PARAMETER_GATES + _CLIFFORD_T_GATES]
+        basis_set = [op.__name__ for op in _PARAMETER_GATES + _CLIFFORD_T_GATES + _SKIP_OP_TYPES]
```

Comment:
This keeps the operators that belong in `_SKIP_OP_TYPES` from not being disregarding in `compile` as they are now a part of the `basis_set` to compile down to.